cpp/src/barretenberg/honk/proof_system/prover.cpp

```diff
@@ -217,13 +217,14 @@ template <StandardFlavor Flavor> void StandardProver_<Flavor>::execute_shplonk_p
 }
 
 /**
- * - Compute KZG quotient commitment [W]_1.
- *
+ * - Compute final PCS opening proof:
+ * - For KZG, this is the quotient commitment [W]_1
+ * - For IPA, the vectors L and R
  * */
-template <StandardFlavor Flavor> void StandardProver_<Flavor>::execute_kzg_round()
+template <StandardFlavor Flavor> void StandardProver_<Flavor>::execute_final_pcs_round()
 {
-    quotient_W = KZG::compute_opening_proof_polynomial(shplonk_output.opening_pair, shplonk_output.witness);
-    queue.add_commitment(quotient_W, "KZG:W");
+    PCS::compute_opening_proof(pcs_commitment_key, shplonk_output.opening_pair, shplonk_output.witness, transcript);
+    info("done");
```

Comment:
Do we need the info?

Reply:
No, I have temporarily taken out the work_queue from KZG to have the interfaces unifies and left a TODO in KZG to put it back when we integrate it in IPA as well. Adding a TODO here as well.